src/components/themed/OutlinedTextInput.tsx

```diff
@@ -326,12 +319,14 @@ export const OutlinedTextInput = React.forwardRef<OutlinedTextInputRef, Outlined
   const charLimitLabel = maxLength === undefined ? '' : `${maxLength - value.length}`
 
   const numpad = props.keyboardType === 'decimal-pad'
-  const textStyle = numpad ? styles.numberInput : styles.textInput
-  const suffixStyle = React.useMemo(() => [styles.suffixText, Platform.OS === 'android' ? styles.suffixAndroidAdjust : null], [styles])
```

Comment:
Does this work properly on android?

Reply:
After extensive testing, I got it to work pretty well on both platforms.